package.json

```diff
@@ -102,24 +102,35 @@
     ],
     "rules": {
       "angular/angularelement": "error",
-      "angular/di": "error",
-      "angular/di-order": "error",
+      "angular/di": "warn",
+      "angular/di-order": "warn",
       "angular/di-unused": "error",
       "angular/document-service": "error",
       "angular/log": "error",
       "angular/module-dependency-order": "error",
       "angular/no-service-method": "error",
-      "angular/no-services": "error",
+      "angular/no-services": "warn",
       "angular/window-service": "error",
       "compat/compat": "error",
       "arrow-parens": "off",
       "constructor-super": "off",
       "generator-star-spacing": "off",
+      "max-len": [
+        "error",
+        {
+          "code": 120,
+          "tabWidth": 2,
+          "ignoreUrls": true
+        }
+      ],
       "no-new-symbol": "off",
       "no-this-before-super": "off",
+      "no-invalid-this": "off",
```

Comment:
It's a mixed bag throughout the project. My personal preference is to use controllerAs and the `var vm = this`, avoiding `$scope` and `this` entirely (something I picked up from the [johnpapa Angular style guide](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md). This project is a long, long way from that though. 

In general I'm in favor of avoiding `this` inside angular controllers.